lua/load_wezterm_nvim.lua

```diff
@@ -0,0 +1,97 @@
+-- loads the wezterm.nvim plugin and exposes methods to the python remote plugin
+local ok, wezterm = pcall(require, "wezterm")
+if not ok then
+  vim.api.nvim_err_writeln("[Molten] `wezterm.nvim` not found")
+  return
+end
+
+local wezterm_api = {}
+
+wezterm_api.get_pane_id = function()
+  local current_pane_id = wezterm.get_current_pane()
+  return current_pane_id
+end
+
+--- Validate the split direction
+--- type function
+--- @param direction string the direction to validate
+--- @return string validated direction if valid
+local validate_split_dir = function(direction)
+  local accepted_dirs = { "top", "bottom", "left", "right" }
+  --if direction not in accepted_dirs, return "bottom" else return direction
+  if not vim.tbl_contains(accepted_dirs, direction) then
+    vim.api.nvim_err_writeln(
+      "'molten_split_dir' must be one of 'top', 'bottom', 'left', or 'right', defaulting to 'right'"
+    )
+    return "right"
+  end
+  return direction
+end
+
+--- Validate the split size
+--- type function
+--- @param size number the size to validate
+--- @return number validated size if valid
+local validate_split_size = function(size)
+  if size < 0 or size > 100 or size == nil then

```

Comment:
nil check needs to come first, otherwise this will throw an error trying to compare nil to a number.